src/pages/settings/tabs/currencies/edit-currency.vue

```diff
@@ -190,6 +243,19 @@ export default defineComponent({
   gap: 32px;
   margin-top: 32px;
 }
+.edit-currency__checkbox {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  .edit-currency__label {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .live-span {
+      margin-right: 10px;
+    }
+  }
+}
```

Comment:
As you might saw in the app there's no nesting used for CSS classes. Nesting is used for pseudo-elements or pseudo-classes only.